.npmignore

```diff
@@ -4,5 +4,6 @@
 .flowconfig
 .travis.yml
 src/
+tests/
```

Comment:
Can you also add the `**/*.spec.js` and `**/*.spec.js.flow` patterns to also prevent the unit tests files to be published?

Reply:
Added!